src/Tui.hs

```diff
@@ -98,12 +102,21 @@ msgSchema = MessageSchema { msgSchemaName = MN "SearchRequest"
                             }]
                           }
 
-processFTV :: String -> String -> FieldValue
+data FieldChoice = FReq | FOpt | FRep | Wrong deriving (Show, Read)
+
+toFC :: String -> FieldChoice
+toFC s = case s of
+    "query"    -> FReq
+    "maybeNum" -> FOpt
+    "numList"  -> FRep
+    _          -> Wrong
```

Comment:
I've created issue to change behavior of this function in future:

* https://github.com/Holmusk/proto-route/issues/15